string roi_id
---
bool success